addons/survey/static/src/js/survey_form.js

```diff
@@ -442,7 +442,9 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend({
             resolveFadeOut();
         });
         // Background management - Fade in / out on each transition
-        $('div.o_survey_background').addClass('o_survey_background_transition');
+        if (this.options.refreshBackground) {
```

Comment:
Note: 3rd commit: typo "suyrvey" -> "survey"

Would also say "if no background is set on any survey section, the surveys behaves exactly like before introduction ..." .